public/kbn_sankey_vis_controller.js

```diff
@@ -115,7 +128,7 @@ module.controller('KbnSankeyVisController', function ($scope, $element, $rootSco
         return d.color;
       })
       .style('stroke', function (d) {
-        return d3.rgb(d.color).darker(2);
+        return (IS_DARK_THEME) ? d3.rgb(d.color).brighter(1) : d3.rgb(d.color).darker(1);
```

Comment:
Hi @mmguero , I think changing the old parameter `darker(1)` to  `darker(2)` is unnecessary .

Reply:
Okay, that's fine. I will revert that also.